nservicebus/messaging/transactions.md

```diff
@@ -7,94 +7,106 @@ tags:
 
-snippet:CustomTransactionTimeout
+WARNING: In version 5 and below, when transactions are disabled, no retries will be performed and messages will not be forwarded to the error queue in the event of any failure.
 
-Or via your *.config file see an example [here](https://msdn.microsoft.com/en-us/library/system.transactions.configuration.defaultsettingssection%28v=vs.100%29.aspx#Anchor_5). 
+snippet:TransactionsDisable
 
+### Controlling transaction scope options
 
-## Handlers
+NServiceBus allows you to control the following options for transaction scopes created when processing messages.
 
-In the ambient transaction mode NServiceBus executes all message handlers in a single `TransactionScope` in order to ensure that all the data manipulations are either executed as a whole or rolled back as a whole. This behavior can be disabled using the following
+#### Isolation level
 
-<!-- import TransactionsDoNotWrapHandlersExecutionInATransactionScope --> 
+NServiceBus will by default use the `ReadCommitted` [isolation level](https://msdn.microsoft.com/en-us/library/system.transactions.isolationlevel).
 
-This results in suppressing any ambient transaction that exist. This effectively turns the *ambient transaction* mode into *transport transaction* mode with regards to handler behavior (the only difference is the type of transaction used by the transport). Some (or all) handlers might get invoked multiple times and partial results might be visible: 
+NOTE: Version 3 and below used the default isolation level of .Net which is `Serializable`.
 
- * partial updates (where one handler succeeded updating its data but the other didn't), 
- * partial sends (where some of the messages has been sent but others not),
- * sends without matching updates (where messages has already been sent but the update failed).
+You can change the isolation level using
 
+snippet:CustomTransactionIsolationLevel
 
-NOTE: Starting with version 5, in the *transport transaction* and *unreliable* modes this behavior is the default. Wrapping the message handlers with `TransactionScope` has to be enabled explicitly.
+#### Transaction timeout
 
-snippet:TransactionsWrapHandlersExecutionInATransactionScope
+NServiceBus will use the [default transaction timeout](https://msdn.microsoft.com/en-us/library/system.transactions.transactionmanager.defaulttimeout) of the machine the endpoint is running on.
 
-## Outbox
+You can change the transaction timeout using
+
+snippet:CustomTransactionTimeout
 
-Outbox is a [feature](/nservicebus/outbox)  that enhances the *transport transaction* mode guarantees. 
+Or via your .config file see an example [here](https://msdn.microsoft.com/en-us/library/system.transactions.configuration.defaultsettingssection%28v=vs.100%29.aspx#Anchor_5).
 
-snippet:TransactionsOutbox
+## Outbox
```

Comment:
No note about Outbox requiring storage in the same database as business data in order to piggyback on the native transaction there?